app/scripts/contentscript.js

```diff
@@ -358,12 +358,15 @@ function blockedDomainCheck() {
 
 /**
  * Redirects the current page to a phishing information page
+ *
+ * @param data
  */
-function redirectToPhishingWarning() {
+function redirectToPhishingWarning(data) {
```

Comment:
We could do `data = {}` and remove the `?.` from `newIssueUrl` below.

Reply:
Sure!

Reply:
Done [here](https://github.com/MetaMask/metamask-extension/pull/14906/commits/b92157093570825805d4034612199e0efaab8d24)!